pkg/kubectl/cmd/describe.go

```diff
@@ -31,22 +32,32 @@ import (
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/runtime"
 )
 
-func NewCmdDescribe(f *cmdutil.Factory, out io.Writer) *cobra.Command {
-	cmd := &cobra.Command{
-		Use:   "describe (RESOURCE NAME | RESOURCE/NAME)",
-		Short: "Show details of a specific resource",
-		Long: `Show details of a specific resource.
+const (
+	describe_long = `Show details of a specific resource.
 
 This command joins many API calls together to form a detailed description of a
-given resource.`,
-		Example: `// Describe a node
+given resource.
+
+Possible resources include pods (po), replication controllers (rc), services
```

Comment:
@jlowdermilk - Sure, this message is copied from get command. I opened a PR #10743 a few days ago to fix small errors in this message for get command, but it is not reviewed yet. I think same changes should be made to #10743. I know you're busy these days, but since these two PR are related, can you also take a look at #10743?